### PR TITLE
Issue #13213: Remove ok comments from Fallthrough Package

### PR DESCRIPTION
**Pull request by Rohanraj123**
part of #13213 

Proof that supressions have removed : 
rajro@DESKTOP-HSV42DL MINGW64 ~/checkstyle (master)
```
$ grep fallthrough config/checkstyle-input-suppressions.xml
            files="checks[\\/]coding[\\/]fallthrough[\\/]InputFallThrough.java"/>
            files="checks[\\/]coding[\\/]fallthrough[\\/]InputFallThrough3.java"/>
            files="checks[\\/]coding[\\/]fallthrough[\\/]InputFallThrough3.java"/>
            files="checks[\\/]coding[\\/]fallthrough[\\/]InputFallThrough3.java"/>
            files="checks[\\/]coding[\\/]fallthrough[\\/]InputFallThrough4.java"/>
            files="checks[\\/]coding[\\/]fallthrough[\\/]InputFallThrough.java"/>
            files="checks[\\/]coding[\\/]fallthrough[\\/]InputFallThrough.java"/>
            files="checks[\\/]coding[\\/]fallthrough[\\/]InputFallThrough3.java"/>
            files="checks[\\/]coding[\\/]fallthrough[\\/]InputFallThrough3.java"/>
            files="checks[\\/]coding[\\/]fallthrough[\\/]InputFallThrough5.java"/>
            files="checks[\\/]coding[\\/]fallthrough[\\/]InputFallThrough7.java"/>
            files="checks[\\/]coding[\\/]fallthrough[\\/]InputFallThrough7.java"/>
            files="checks[\\/]coding[\\/]fallthrough[\\/]InputFallThroughDefault.java"/>
            files="checks[\\/]coding[\\/]fallthrough[\\/]InputFallThroughDefault.java"/>
            files="checks[\\/]coding[\\/]fallthrough[\\/]InputFallThroughFallThroughLotsOfComment2.java"/>
            files="checks[\\/]coding[\\/]fallthrough[\\/]InputFallThroughStringSwitch.java"/>
            files="checks[\\/]coding[\\/]fallthrough[\\/]InputFallThroughWithEmoji.java"/>
            files="checks[\\/]coding[\\/]fallthrough[\\/]InputFallThroughWithEmoji.java"/>
            files="checks[\\/]coding[\\/]fallthrough[\\/]InputFallThroughWithEmoji.java"/>

rajro@DESKTOP-HSV42DL MINGW64 ~/checkstyle (master)
$ git checkout fallthrough
Switched to branch 'fallthrough'

rajro@DESKTOP-HSV42DL MINGW64 ~/checkstyle (fallthrough)
$ grep fallthrough config/checkstyle-input-suppressions.xml

rajro@DESKTOP-HSV42DL MINGW64 ~/checkstyle (fallthrough)
$ echo $?
1

rajro@DESKTOP-HSV42DL MINGW64 ~/checkstyle (fallthrough)
$

```

### CODE REVIEW

**Comment by Rohanraj123** [comment]
@romani please put `hacktoberfest-accepted` label

**Review by romani — APPROVED** [review]
Ok to merge

**Comment by Rohanraj123** [comment]
> Ok to merge

Can we please put Hacktoberfest-accepted label over PR.

**Comment by romani** [comment]
Please pull latest sources to your fork.
Pay attention to failing CI for age of your code.

No need to rebase this PR

**Comment by Rohanraj123** [comment]
> Please pull latest sources to your fork. Pay attention to failing CI for age of your code.
> 
> No need to rebase this PR

Did it. and now CI checks are passing.

File: src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough.java [hunk 1]
```diff
@@ -43,7 +43,7 @@ void tryResource() throws Exception {
                 return;
             }
             finally {
-            } // ok
```
Comment (nrmancuso) [inline src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough.java:46]: This is a good one to keep, please do something like `// ok, either exception is thrown or we return from method before fallthrough can occur`

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Minor item:

File: src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough.java (hunk 1 above)
Comment (Rohanraj123) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough.java:46]: done

**Review by nrmancuso — APPROVED** [review]

**Pull request merged**